it/uk/gov/hmrc/excisemovementcontrolsystemapi/DraftExciseMovementControllerItSpec.scala

```diff
@@ -99,6 +158,16 @@ class DraftExciseMovementControllerItSpec extends PlaySpec
     }
   }
 
+  private def createEISErrorResponseBody(message: String) = {
+    Json.toJson(EISErrorResponse(
+      LocalDateTime.of(2023, 12, 5, 12, 5, 6),
+      message,
+      "bad request",
```

Comment:
Yes this can be capitalise

Reply:
actually this it does not matter in the test as the test s just testing that we are getting is the same message that the EIS give us, what ever this may be (which we stub)